tests/integration/cartography/intel/aws/test_route53_sync.py

```diff
@@ -317,7 +318,7 @@ def test_sync_route53_cleanup(mock_get_zones, neo4j_session):
 @patch.object(
     cartography.intel.aws.route53,
     "get_zones",
-    return_value=GET_ZONES_SAMPLE_RESPONSE,
```

Comment:
[question] I'm confused about why we need to change the test this much. Was the data or the setup incorrect to begin with?